skipper.go

```diff
@@ -985,7 +990,16 @@ func listen(o *Options, mtr metrics.Metrics) (net.Listener, error) {
 	}
 
 	if !o.EnableTCPQueue {
-		return net.Listen("tcp", o.Address)
+		ln, err := net.Listen("tcp", o.Address)
+		if o.ProxyProtocolEnable {
+			log.Printf("Proxy protocol listener")
```

Comment:
Same thing on the log goes here. Check other logs in this file. I see that they're using `log.Info` in this file.